js/index.js

```diff
@@ -1,65 +1,158 @@
-    if (x.style.display === "none") {
-      x.style.display = "block";
-    } else {
-      x.style.display = "none";
-    }
-  } 
-  
-  function myFunction2() {
-    var x = document.getElementById("miru-mir");
-    if (x.style.display === "none") {
-      x.style.display = "block";
-    } else {
-      x.style.display = "none";
-    }
-  } 
-  
-  function myFunction3() {
-    var x = document.getElementById("alter-ego");
-    if (x.style.display === "none") {
-      x.style.display = "block";
-    } else {
-      x.style.display = "none";
-    }
-  } 
-  
-  function myFunction4() {
-    var x = document.getElementById("hansung");
-    if (x.style.display === "none") {
-      x.style.display = "block";
-    } else {
-      x.style.display = "none";
-    }
-  } 
-  
-  function myFunction5() {
-    var x = document.getElementById("variant-optima");
-    if (x.style.display === "none") {
-      x.style.display = "block";
-    } else {
-      x.style.display = "none";
-    }
-  } 
-  
-  function myFunction6() {
-    var x = document.getElementById("school");
-    if (x.style.display === "none") {
-      x.style.display = "block";
-    } else {
-      x.style.display = "none";
-    }
-  } 
-  
 /* Toggle between adding and removing the "responsive" class to topnav when the user clicks on the icon */
-function myFunction() {
+function hamburger() {
   var x = document.getElementById("myNav");
   if (x.className === "topnav") {
     x.className += " responsive";
   } else {
     x.className = "topnav";
   }
-} 
\ No newline at end of file
+  } 
+
+
+// Experience section //
+
+function myFunction1() {
+  var x = document.getElementById("tjx");
+  if (x.style.display === "none") {
+    x.style.display = "block";
+  } else {
+    x.style.display = "none";
+  }
+} 
+
+function myFunction2() {
+  var x = document.getElementById("miru-mir");
+  if (x.style.display === "none") {
+    x.style.display = "block";
+  } else {
+    x.style.display = "none";
+  }
+} 
+
+function myFunction3() {
+  var x = document.getElementById("alter-ego");
+  if (x.style.display === "none") {
+    x.style.display = "block";
+  } else {
+    x.style.display = "none";
+  }
+} 
+
+function myFunction4() {
+  var x = document.getElementById("hansung");
+  if (x.style.display === "none") {
+    x.style.display = "block";
+  } else {
+    x.style.display = "none";
+  }
+} 
+
+function myFunction5() {
+  var x = document.getElementById("variant-optima");
+  if (x.style.display === "none") {
+    x.style.display = "block";
+  } else {
+    x.style.display = "none";
+  }
+} 
+
+function myFunction6() {
+  var x = document.getElementById("school");
+  if (x.style.display === "none") {
+    x.style.display = "block";
+  } else {
+    x.style.display = "none";
+  }
+} 
+
+/*
+function myFunction(element, id) {
+
+  var x = document.getElementById();
+  
+  if (x.style.display === "none") {
+    x.style.display = "block";
+  } else {
+    x.style.display = "none";
+  }
+} 
+*/
```

Comment:
You've almost got it here. Since you're passing the element in and you need to get the element by id, it should be as simple as calling `document.getElementId()` using the passed in element. And with that, the 6 functions above can be replaced.

My only other suggestion  would be to choose a more descriptive name for this function.